app/models/foreman_openscap/arf_report.rb

```diff
@@ -13,8 +13,13 @@ class ArfReport < ::Report
     has_one :policy, :through => :policy_arf_report
     has_one :asset, :through => :host, :class_name => 'ForemanOpenscap::Asset'
     after_save :assign_locations_organizations
+    has_one :log, :foreign_key => :report_id
+
+    delegate :result, :to => :log, :allow_nil => true
```

Comment:
I have absolutely no clue how this worked until now, as there is no result field on the report. the validation should have thrown an exception in rails 3 as well...